service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/HmctsEmailContentProvider.java

```diff
@@ -30,4 +46,35 @@ public HmctsEmailContentProvider(LocalAuthorityNameLookupConfiguration localAuth
             .put("localAuthority", localAuthorityNameLookupConfiguration.getLocalAuthorityName(localAuthorityCode))
             .build();
     }
+
+    public Map<String, Object> buildC2UploadNotification(final CaseDetails caseDetails,
+                                                         final String localAuthorityCode) {
+        // Validation within our frontend ensures that the following data is present
+        CaseData caseData = mapper.getObjectMapper().convertValue(caseDetails.getData(), CaseData.class);
+
+        List<Map<String, Object>> respondents1 = mapper.getObjectMapper().convertValue(
+            caseDetails.getData().get("respondents1"), new TypeReference<>() {});
+
+        List<Respondent> respondents = respondents1.stream()
+            .map(respondent ->
+                mapper.getObjectMapper().convertValue(respondent.get("value"), Respondent.class))
+            .collect(toList());
+
+        return Map.of(
+            "court", hmctsCourtLookupConfiguration.getCourt(localAuthorityCode).getName(),
+            "lastNameOfRespondent", respondents
```

Comment:
this will only ever get the lastName of the first entry in respondents1 right? Is this correct?

Reply:
currently yes, as per the jira.

Reply:
discussed, need to investigate with BA

Reply:
Mia, has confirmed we will need the first respondent.